tests/test_app.py

```diff
@@ -280,25 +283,52 @@ def test_version_p(self) -> None:
     def test_patch_save_json_p(self) -> None:
         target_path = str(SAMPLES_DIR / "password.patch")
         with tempfile.TemporaryDirectory() as tmp_dir:
-            json_filename = os.path.join(tmp_dir, "unittest_output.json")
+            json_filename = os.path.join(tmp_dir, f"{__name__}.json")
             _stdout, _stderr = self._m_credsweeper(
                 ["--diff_path", target_path, "--save-json", json_filename, "--log", "silence"])
-            assert os.path.exists(os.path.join(tmp_dir, "unittest_output_added.json"))
-            assert os.path.exists(os.path.join(tmp_dir, "unittest_output_deleted.json"))
+            assert os.path.exists(os.path.join(tmp_dir, f"{__name__}_added.json"))
+            assert os.path.exists(os.path.join(tmp_dir, f"{__name__}_deleted.json"))
 
     # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
     def test_patch_save_json_n(self) -> None:
         target_path = str(SAMPLES_DIR / "password.patch")
         _stdout, _stderr = self._m_credsweeper(["--diff_path", target_path, "--log", "silence"])
-        assert not os.path.exists(os.path.join(PROJECT_DIR, "unittest_output_added.json"))
-        assert not os.path.exists(os.path.join(PROJECT_DIR, "unittest_output_deleted.json"))
+        for root, dirs, files in os.walk(PROJECT_DIR):
+            self.assertIn('credsweeper', dirs)
+            for file in files:
+                self.assertFalse(file.endswith('.json'))
+            break
```

Comment:
In my opinion using break and this for loop is not that improve the code than before.
How do you think about it?